app/components/ui/atoms/CalloutTextBox.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Box } from 'grid-styled'
+import { th } from '@pubsweet/ui-toolkit'
+
+const Frame = styled(Box)`
+  border: ${th('borderWidth')} ${th('borderStyle')} ${th('colorPrimary')};
+  background-color: ${th('colorPrimary')}20;
+`
+
+const CalloutTextBox = ({ onClose, children, ...props }) => (
```

Comment:
I think `onClose` might have been added accidentally here. Perhaps @tamlyn was originally copy/pasting from our `CalloutBox` file, but there is no open/close functionality associated with this component.